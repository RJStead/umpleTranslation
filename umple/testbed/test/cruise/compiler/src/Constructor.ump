generate Java "../src-gen-umple";

class Constructor
{
    Integer a;
    someString;
    
    public Constructor(Integer x, String s) {
        someString = "";
        for (int i = 0; i < x; i ++) {
            someString += s;
        }
        a = s;
    }
}
