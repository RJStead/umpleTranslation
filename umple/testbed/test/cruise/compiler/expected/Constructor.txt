Warning 170 on line 8 of file 'Constructor.ump':
Custom constructors should normally not be provided in Umple. Consider using before or after keywords to adjust what the constructor does, or adjusting constructor parameters using keywords like lazy.
Constructor.ump:13: error: incompatible types: String cannot be converted to int
        a = s;
            ^
1 error
